sphinx/source/_templates/layout.html

```diff
@@ -14,40 +14,6 @@
     <link rel="icon" type="image/png" sizes="16x16" href="https://static.bokeh.org/favicon/favicon-16x16.png">
 {%- endblock %}
 
-{% block docs_sidebar %}
-  <!-- Only show if we have sidebars configured -->
-  {% if sidebars %}
-  {{ super() }}
-  {% endif %}
-{% endblock %}
-
-{% block docs_toc %}
-  <!-- Only show if we have sidebars configured -->
-  {% if sidebars %}
-  {{ super() }}
-  {% endif %}
-{% endblock %}
-
-{% block docs_main %}
-{% if sidebars %}
-  {% set content_col_class = "col-md-9 col-xl-7" %}
-{% else %}
-  {% set content_col_class = "col-md-12" %}
```

Comment:
@tcmetzger FYI I forgot to mention, this is the one difference I did notice. In the official theme this is `"col-md-11 col-xl-8"` which is probably fine.

Reply:
I think this is a customization that I made for the docs landing page and the gallery page (where we don't have sidebars). This is what the landing page looks like with the customization in place (current 2.4 docs):
![image](https://user-images.githubusercontent.com/39711796/136141349-4285b806-f37e-491a-a2ec-ca57c0d23cd2.png)

And this is what the landing page looks like in my local build of this branch, i.e. without the customization (the main content is now squeezed into a narrower space):
![image](https://user-images.githubusercontent.com/39711796/136141404-dfeaa85f-bfe8-41b7-b3c6-68e2a4bbdd6c.png)

I definitely prefer the version with the customization in place. 

I'll do some digging to see if we can get a similar result without making this change in `layout.html`!

Reply:
@tcmetzger I also prefer the way we had it with the wider view. I think it will affect gallery pages too, which should also fill width. I think we may have to put the customization back, in which case we will just have to update it to match the current `docs_main` block, except with the one change. Alternatively, there may be a way to override the default template used, per page, I'm not sure.

@choldgraf any thoughts if/how we might accomplish this without restoring the manual template customization?

Reply:
Actually unfortunately it seems like we need the `docs_sidebar` and `docs_toc` customizations too :/

Reply:
well, the extra sidebar space can be fixed with CSS so I will do that for now.